### ISSUE TITLE
Stops updating

### ISSUE THREAD

**Issue by themarkymark-steem** [elgatosf/streamdeck-cpu#2]
No longer updates and just displays outside blue border with no numbers.
Quitting the StreamDeck app and restarting seems to fix it for a short time.

Windows 10 x64 Pro
Latest StreamDeck software

**Comment by Timac** [comment]
Could you please try Stream Deck 4.1 which addresses several issues with plugins. This new version might have solved the problem you reported.

**Comment by themarkymark-steem** [comment]
I am using 4.1, and it does seem to be a lot better.  I have not seen it yet, but still testing.

**Comment by themarkymark-steem** [comment]
Still a problem.

![image](https://user-images.githubusercontent.com/33408375/54356089-1f101400-4631-11e9-8bc5-7f42c4879fa2.png)

This time I noticed when it happened.   I rebooted my machine and when it came back up it was in this state.

I tried deleting the action from my bar and readding it, and same issue.

I went to uninstall it (which is how I got it working again last time) and streamdeck exited, when I relaunched Streamdeck manually, I looked to reinstall it, it was still installed.  I decide to add it to my bar again and this time it is working again.
